test/controllers/users/sessions_controller_test.rb

```diff
@@ -92,7 +92,7 @@ class Users::SessionsControllerTest < ActionDispatch::IntegrationTest
         password: '123456'
       }
     }
-    post users_sign_in_url, params: payload
+    post user_session_url, params: payload
```

Comment:
@ajaystha42 why are these redirects different now?

Reply:
@donrestarone  we are overriding the method of gem so I believe thats the reason we need to redirect it to new url